assets/js/util.ts

```diff
@@ -185,12 +181,13 @@ export const placesWithSelectedAlert = (
     : [];
 };
 
-export const signsByZone = (signs: Screen[]) => {
+export const signsByDirection = (signs: Screen[]) => {
   const groupedSigns = fp.groupBy((sign) => {
-    if (fp.includes(sign.zone, LEFT_ZONES)) {
-      return "left";
-    } else if (fp.includes(sign.zone, MIDDLE_ZONES)) {
+    const directionIds = sign.routes?.map((r) => r.direction_id);
+    if (fp.intersection(directionIds, [0, 1]).length === 2) {
       return "middle";
+    } else if (directionIds?.includes(0)) {
+      return "left";
```

Comment:
I'm a little confused here. Under this logic if a sign has direction `0` but not direction `1`, it goes in the `left` group. But elsewhere, the `left` selection button appears to select all signs with both directions.

Reply:
Your description sounds right and is by design. The grouping buttons select signs with the labeled direction so `middle` signs by default will always be selected when either button is clicked.

Reply:
Oh, duh, the grouping of the columns is [0 / both / 1], but the arrangement of the buttons is [both / 0 / 1]. Not sure why I was confused there 😅